data/events/2022-zurich.yml

```
@@ -37,8 +37,8 @@ location_address: "Technikumstrasse 8, 8400 Winterthur, Switzerland" #Optional -
 
 nav_elements: # List of pages you want to show up in the navigation of your page.
   - name: propose
-#   - name: program
-#   - name: speakers
+#  - name: program
```

```suggestion
  - name: program
```

Uncomment this and the speakers to ensure these appear in the navigation bar

---

We have not added them to the navigation yet on purpose as we are still finalizing.